tests/e2e/test_browser_coverage.py

```
@@ -0,0 +1,108 @@
+"""Every registry browser×OS pair must appear in the claimed-browser matrix."""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)